src/views/Auth/SignIn.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import { Typography, TextField, Container } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+import { Button } from "@material-ui/core";
+import Icon from "@material-ui/core/Icon";
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: "20%",
+  },
+  form: {
+    width: "100vw",
+  },
+  textfield: {
+    width: "40%",
+    display: "block",
+  },
+  btn: {
+    width: "15%",
+    borderRadius: "20px",
+  },
+});
+
+type FormData = {
+  email: string;
+  password: string;
+};
+
+const SignIn: React.FC = () => {
+  const [name, setName] = useState("");
+  const { register, handleSubmit } = useForm<FormData>();
+  const classes = useStyles();
+
+  const onSubmit = handleSubmit(({ email, password }) => {
+    console.log(email, password);
+  });
+
+  return (
+    <Container className={classes.root}>
+      <Typography variant="h3">Get started for free</Typography>
```

Comment:
Tutaj ten text `Get started for free` do constów daj, tak jak z navbarem

Reply:
Tutaj też będziesz musiała dodać folder w auth -> consts i dopiero w folderze consts dasz plik np signPage.const.ts